templates/base.html

```diff
@@ -12,6 +12,6 @@
 
```

Comment:
hmm, not allowed to leave a comment above line 12 but I think we need to include  the meta tag `<meta name="viewport" content="width=device-width, initial-scale=1.0">` in the `<head>` to set the viewport -- [ more info here ](https://www.w3schools.com/css/css_rwd_viewport.asp). Otherwise the browser is not reading properly the media queries we provide.

Reply:
opps sorry missed this, will add in the next PR :)